sound/soc/sof/intel/hda-ipc.c

```diff
@@ -107,8 +107,8 @@ void hda_dsp_ipc_get_reply(struct snd_sof_dev *sdev)
 	} else {
 		/* reply correct size ? */
 		if (reply.hdr.size != msg->reply_size &&
-			/* getter payload is never known upfront */
-			!(reply.hdr.cmd & SOF_IPC_GLB_PROBE)) {
+		    /* getter payload is never known upfront */
+		    ((reply.hdr.cmd & SOF_GLB_TYPE_MASK) != SOF_IPC_GLB_PROBE)) {
```

Comment:
if you use fix in the commit message should there be a  Fixes tag?

````
Fixes: f3b433e4699f  ('ASoC: SOF: Implement Probe IPC API')
````

````

Reply:
Thanks, added

Reply:
@lyakh do you mind changing the prefix to ASoC; SOF: Intel: hda-ipc: fix reply size checking?